utils/time.js

```diff
@@ -0,0 +1,24 @@
+const hrTimeSupport = typeof process !== 'undefined' && process.hrtime;
```

Comment:
I _think_ this could be simplified to:
```suggestion
const hrTimeSupport = process && process.hrtime;
```